docs/source/http_services.rst

```diff
@@ -49,6 +49,22 @@ Pages and JSON endpoints served by the scheduler
 - ``/statics/()``: static file content (CSS, etc)
 - ``/stealing``: worker occupancy metrics, to evaluate task stealing
 
+Scheduler API
+-------------
+
+Scheduler methods exposed by the API with an example of the request body they take
+
+- ``/api/v1/retire_workers`` : retire certain workers on the scheduler
```

Comment:
It would be nice if this mentioned specifically that _addresses_ are expected and not names. Sometimes they can be interchangable but this API expects addresses. The example below might still be misleading since we're setting names, by default, to the addresses.